index.html

```diff
@@ -18,17 +18,23 @@
     <header class="header">
       <nav class="navbar">
         <div class="logo">BZ.</div>
-        <img src="/images/Union.png" alt="" />
+        <img src="images/Union.png" alt="menu icon" />
+        <ul class="nav-list">
+          <li class="nav__item"><a href="#">Portfolio</a></li>
+          <li class="nav__item"><a href="#">About</a></li>
+          <li class="nav__item"><a href="#">Contact</a></li>

```

Comment:
[OPTIONAL]

You could add links to the id's to the corresponding sections here and your social media icons :wink:

Reply:
thanks its very good comment.